fontc/src/lib.rs

```diff
@@ -706,6 +706,7 @@ mod tests {
         fs::{self, File},
         io::Read,
         path::{Path, PathBuf},
+        str::FromStr,
```

Comment:
noob question: why is this needed?

Reply:
A couple of the tests use `Tag::from_str` and the `FromStr` trait needs to be in scope for that method to be available.

Reply:
Interesting. AFAIU, the Tag struct implements `FromStr` trait, which comes from `std::str::FromStr`; what it's not immediately clear to me is why the users of that method need to import that trait, and it is not enough to simply import Tag.

Reply:
Just the rules of the language. The trait itself must be in scope in order to call its methods. You can implement multiple traits with the same method name(s) for a single type and this is one way of disambiguating those cases.

This is sometimes muddled because Rust has a [prelude](https://doc.rust-lang.org/std/prelude/index.html) that automatically imports common types and traits to all modules. So you’ll not see traits imported explicitly if they appear there.